fast_forward/universe_handler.py

```diff
@@ -36,6 +54,19 @@ def __init__(self, mol_names, *args, **kwargs):
         mol_idxs_names = [(idx, mol_name) for idx, mol_name in self.mol_idxs_by_name.items()]
         self.normal_order = mol_idxs_names.sort()
 
+        istraj = args[1].casefold().endswith(('trr', 'xtc'))
+        
+        # Required for unwrapping
+        if istraj:
```

Comment:
This means we always PBC treat the trajectory. Could there be a use-case where we don't want that? Perhaps we should enable by default but still have a flag that let's the user switch it off.

Reply:
The PBC correction never has adverse effects, so the only use would be to speed up the processing a bit, at the cost of complicating the code, which is a conditional on the reciprocal space mapping and having two functions for the mapping.